docs/sphinx/source/fsd/FSD-Appendices.md

```diff
@@ -0,0 +1,30 @@
+| horizontal\_uncert\_var   | Float       | See S-101 horizontal position uncertainty variable factor                                                                        |
+| survey\_date\_start       | String      | See S-101 Survey date start                                                                                                      |
+| survey\_date\_end         | String      | See S-101 Survey date end                                                                                                        |
+| source\_institution       | String      | e.g. "NOAA Office of Coast Survey"                                                                                               |
+| source\_survey\_id        | String      | e.g. "H99999"                                                                                                                    |
+| license\_name             | String      | e.g. "CC0 1.0"                                                                                                                   |
+| license\_url              | String      | A URL or DOI (ideally in URL form) referencing the license definition, e.g. "https://creativecommons.org/publicdomain/zero/1.0/" |
+
+#### Further discussion
+
+1. feature_size_var is meant to augment feature_size which corresponds to S-101 size of features detected. As noted in S-101, size of features detected is intended to be described as the smallest size in cubic metres the survey was capable of detecting. Depending on the type of survey this definition might force different depth ranges to have different values. For example, a survey vessel that works at a fixed height off the seafloor could maintain a fixed feature detection size capability over a wide range of depths. A surface vessel working over those same range of depths may have a feature detection capability that varies with depth causing the detection capability to be ambiguous and potentially misrepresented. For this reason feature_size_var is the percentage of depth that a feature of such size could be detected. When both feature_size and feature_size_var are present the greater of the two should be considered valid. The expectation is that feature_size_var will be set to zero if the feature size does not scale with depth. As with feature_size, feature_size_var should be ignored if significant_features is False.
+
+2. Coast Survey often uses side scan to detect features in flat seafloor areas, thus these surveys have coverage that do not contain direct depth measurements. In these cases the nodes with survey coverage but without bathymetry would be set to False. A condition with coverage = True and bathy_coverage = False is a useful indicator for how to work with these nodes within our workflow. If coverage is False, bathy_coverage must also be False.
```

Comment:
Is it relevant to mention "Coast Survey" here? What about "When side scan is used ..."?

Reply:
@GlenRice-NOAA may have an opinion on this.

Reply:
@GlenRice-NOAA

Reply:
"When side scan is used..." sounds fine to me.  I just wrote it from the Coast Survey perspective because we are the ones creating the profile.

Reply:
Resolved in 923d8dc23c334c2dff9ce43010d5fd6e2dc4e859.